zipline/utils/calendars/trading_calendar.py

```diff
@@ -512,6 +512,34 @@ def sessions_in_range(self, start_session_label, end_session_label):
             )
         ]
 
+    def get_session_indices(self, session_label, offset):
```

Comment:
why does this need to take an offset. To me, a more natural API is `get_session_index`. You can add the offset to that yourself.